<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicClass</class>
 <widget class="QMainWindow" name="musicClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音乐播放器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/cbe12a01b3f672c53afee65d350f86ef.jpeg</normaloff>icon/cbe12a01b3f672c53afee65d350f86ef.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/icon/f83d097ec2b95c80fe22ab3078c96114.jpeg)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="3">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="beforeButton">
       <property name="text">
        <string>上一首</string>
       </property>
      </widget>
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>下一首</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="nowTime">
      <property name="text">
       <string>0:0:0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="local_Music_Button">
      <property name="text">
       <string>本地音乐</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QListWidget" name="musicList">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="online_Music_Button">
      <property name="text">
       <string>在线音乐</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="allTime">
      <property name="text">
       <string>0:0:0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" rowspan="2">
     <widget class="QSlider" name="verticalSlider">
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="music.qrc"/>
 </resources>
 <connections/>
</ui>
